tracker_blackbox_test.go

```diff
@@ -20,6 +21,7 @@ func getTrackerTestData(t *testing.T) []testSecureAPI {
 		t.Fatal("Could not parse Key ", err)
 	}
 	differentPrivatekey, err := jwt.ParseRSAPrivateKeyFromPEM(([]byte(RSADifferentPrivateKeyTest)))
+	require.Nil(t, err)
```

Comment:
Okay, just to be sure. require will stop the execution. Is that intended ? I believe so..